questionnaire-en.json

```diff
@@ -134,21 +134,12 @@
         },
         {
           "correctAnswer": "Yes",
-          "question": "Do you have a process for reviewing open source license obligations, restrictions and rights?",
+          "question": "Do you have a documented procedure to review and document Open Source license obligations, restrictions and rights?",
           "number": "1.i",
           "type": "YES_NO",
           "specReference": [
             "1.5"
           ]
-        },

```

Comment:
By removing a question, this will need to be a major release update to the questionnaire.

Reply:
Is this due to the way the web app works?

Reply:
> Is this due to the way the web app works?

Yes and no - if we make such a substantive change to the questionnaire, the previous answers become unmappable and a human needs to re-assess compliance IMHO.